procrastinate/worker.py

```diff
@@ -338,19 +338,11 @@ async def run(self):
         try:
             # shield the loop task from cancellation
             # instead, a stop event is set to enable graceful shutdown
-            await utils.wait_any(asyncio.shield(loop_task), self._stop_event.wait())
-            if self._stop_event.is_set():
-                try:
-                    await asyncio.wait_for(loop_task, timeout=self.shutdown_timeout)
-                except asyncio.TimeoutError:
-                    pass
+            await asyncio.shield(loop_task)
```

Comment:
Is the comment `instead, a stop event is set to enable graceful shutdown` still true? This line doesn't do anything with the `_stop_event` anymore.

Reply:
The stop event is set by calling `stop()` which happens L344.
It explains why the `loop_task` is being shielded. 

Do you recommend a different wording?

Reply:
Never mind, waiting for the event is not necessary here anymore. I didn't quite understand it at the first attempt, but now it seems more clear to me 🙂.